.gitignore

```diff
@@ -6,3 +6,4 @@ demos/*
 demos1
 TODO.md
 npm-debug.log
+lib/
```

Comment:
We shouldn't put lib in .gitignore 
Files inside .gitignore will not be published on npm.